other/materials_designer/create_island_defect.ipynb

```diff
@@ -2,6 +2,10 @@
  "cells": [
```

Comment:
<p>We should add conditions to esse and set parameters from there (and validate)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>